README.md

```diff
@@ -32,7 +32,11 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Knowledge Management
 
+#### [Bubbl.us](https://bubbl.us/) - **Bubbl.us** Is a web-based mind mapping app allows you to build, save, and share up to three mind maps on the free plan.
```

Comment:
That looks good, I would just suggest changing the title#4 format with the #### to a simple '-' following the same structure format as the rest of the document.